auth-api/src/main/java/io/stargate/auth/api/AuthResource.java

```diff
@@ -36,12 +36,17 @@
 
   private static final Logger logger = LoggerFactory.getLogger(AuthResource.class);
 
-  final AuthenticationService authService;
-  private static final boolean shouldEnableUsernameToken =
-      Boolean.parseBoolean(System.getProperty("stargate.auth_api_enable_username_token", "false"));
```

Comment:
I had to refactor this so we can adjust this behaviour in the tests. It's not optimal to have those different constructors but without a config framework that works with a dependency injection mechanism this is the simplest I came up with.